### DISCUSSION TITLE
Time to run not effective (Unraid)

### DISCUSSION THREAD

**Discussion by iamwoz** [meisnate12/Plex-Meta-Manager#370] [Q&A] [answered]
My time to run is set as default within my docker install (0600) but PMM is running at 0300 - date/time/timezone is correct in the container.

![Screenshot_75](https://user-images.githubusercontent.com/85816876/131040152-3a353113-05b8-442c-9787-879810032f48.png)

![Screenshot_76](https://user-images.githubusercontent.com/85816876/131040165-381844ea-b044-4ec3-a579-9cdb2a5ddffc.png)

**Comment by meisnate12** [comment]
default is at 3am

**Reply by iamwoz** [reply]
So this isn't having any effect then? How do I change the run time? Keep up the awesome work, without a doubt my favourite media tool!

![Screenshot_77](https://user-images.githubusercontent.com/85816876/131043536-90915b2b-b090-4709-8458-7eed4df4eef3.png)

**Comment by iamwoz** [comment, marked as answer]
I have sorted this - the Unraid (Community Apps) default docker configuration for time is incorrect:

Time to Run variable is called `time` - should be `PMM_TIME`

Found this documented here: https://github.com/meisnate12/Plex-Meta-Manager/wiki/Docker-Installation

**Reply by meisnate12** [reply]
i need to test out unraid so I can put PMM up there myself
